### PR TITLE
Add dialog to enter a lat/lon value & zoom there

### PR DESCRIPTION
**Pull request by noffle**
At last! Paste in a `latitude, longitude` and have Mapeo zoom there. Combined with https://github.com/digidem/mapeo-desktop/pull/104 this makes it much easier to share locations and find them again.

2017-08-29-192647_1920x1080_scrot

### CODE REVIEW

**Comment by noffle** [comment]
Failing Appveyor and Travis builds appear to be due to something unrelated. :broken_heart:   I'll try and figure out what's up.

**Comment by noffle** [comment]
@gmaclennan request for translations here too, if you're able. :es:

File: id_monkey_patches/id-ui-latlon-display.js [hunk 1]
```diff
@@ -0,0 +1,237 @@
+// A huge copy/paste swath that adds a lat/lon display element to the bottom
```
Comment (karissa) [inline id_monkey_patches/id-ui-latlon-display.js]: Where's it being copied from?

Comment (noffle) [reply id_monkey_patches/id-ui-latlon-display.js]: Oh, right -- that's not very obvious is it?

The whole `id_monkey_patches/` directory is full of overrides of the iD source code (found in `vendor/`). In this case, it's source code from the original `iD.ui` object.

Comment (karissa) [reply id_monkey_patches/id-ui-latlon-display.js]: ah got it thanks :)

**Comment by noffle** [comment]
Latest commit passes on Travis; the PR build is being slow: https://travis-ci.org/digidem/mapeo-desktop/builds/293909042

**Pull request merged**